docs/competition/team-support/battery-shuffler.md

```diff
@@ -1,25 +0,0 @@
----
-original:
-  authors: Richard Barlow
-  url: https://bitbucket.org/richardbarlow/sr-comp-team-support-coord/wiki/Battery_Shuffler
----
-# Battery Shuffler
-
-A battery shuffler is a volunteer that moves batteries, battery chargers, battery charger PSUs and battery charging bags from the arrival desk to the battery charging station on Saturday morning and between the battery charging station and the kit return desk on Sunday afternoon.
-
-## Saturday Morning Shuffling
-
-Teams arrive on Saturday morning between 9:00 and 9:30. When they arrive we take the following items off of them at the arrival desk:
-
- * 2x batteries
- * 1x battery charger
- * 1x battery charger power supply
- * 1x battery charging bag
-
-All of these parts need to be carried from the arrival desk to the battery charging station where they will be used by us to manage the charging of batteries.
-
-The arrival desk will have two blue Really Useful Boxes with 'Returned batteries/chargers/bags' signs on them. The volunteers manning the arrival desk will retrieve these items off of the teams and place them into these boxes. The Battery Shuffler needs to continually carry these boxes up to the battery charging station, empty their contents and then return the empty boxes downstairs. They must repeat this until 10:00. It probably makes sense to get the arrival desk volunteers to fill one box while the other is being taken upstairs.
-
-## Sunday Afternoon Shuffling
```

Comment:
Do we have anything to make up for removing this section?

Reply:
Oops you're right, I'm getting the person responsible for swapping batteries mixed up with the actual shuffler. I'll revert the removal!